bundles/org.openhab.binding.ventaair/README.md

```diff
@@ -0,0 +1,127 @@
+|----------|------------|-------------------------------------------|
+| ipAddress | Y | The IP Address or hostname of the device. |
+| macAddress | Y | The MAC address of the device. |
+| deviceType | Y | Defines the type of device. It is an integer value and its best to use the automatic discovery to obtain it from the device. |
+| pollingTime | N | The time interval in seconds in which the data should be polled from the device, default is 10 seconds. |
+| hash | N | It is a negative integer value and it is used by the device to identify a connection to a client, like the App from the vendor for example. (*) |
+
+(*) I do not know whether there are devices which are restricted to only one client, so I added this parameter to allow the user to set the same value as his App on the phone (can be obtained via sniffing the network).
+However, the LW60-T allows for multiple connections to different clients, identified by different `hash` values at the same time without issues.
+By default the binding uses "-42", so a new ID that is not known to the device and hence it asks for confirmation, see the Discovery section.
+
+Example Thing configuration:
+
+```
+Thing ventaair:lw60t:humidifier [ ipAddress="192.168.42.69", macAddress="f8:f0:05:a6:4e:03", deviceType=4, pollingTime=10, hash=-42]
+```
+
+## Channels
+
+These are the channels that are currently supported:
+
+
+| channel  | type (RO=read-only)   | description                  |
+|----------|--------|------------------------------|
+| power          | Switch | This is the power on/off channel  |
+| fanSpeed       | Number | This is the channel to control the steps for the speed of the fan |
+| targetHumidity | Number | This channel sets the target humidity that should be try to reach by the device  |
+| timer          | Number | This channel sets the power off timer to the set value in hours, i.e. 3 = turn off in 3 hours from now |
+| sleepMode      | Switch | This channel controls the sleep mode of the device (dims the display and slows down the fan) |
+| childLock      | Switch | This is the control channel for the child lock |
+| automatic      | Switch | This is the control channel to start the automatic operation mode of the device |
+| cleanMode      | Switch (RO) | This is the channel that indicates if the device is in the cleaning mode |
+| temperature    | Number:Temperature (RO) | This channel provides the current measured temperature in Celsius or Fahrenheit as configured on the device |
+| humidity       | Number (RO) | This channel provides the humidity measured by the device in percent |
+| waterLevel     | Number (RO) | This channel indicates the water level of the tank where 1 is equal to the yellow "refill tank" warning on the device/App |
+| fanRPM         | Number (RO) | This channel provides the speed of the ventilation fan |
+| timerTimePassed  | Number (RO) | If a timer has been set, this channel provides the minutes since when the timer was started  |
```

Comment:
Same for below.
```suggestion
| timerTimePassed  | Number:Time (RO) | If a timer has been set, this channel provides the minutes since when the timer was started  |
```